### ISSUE TITLE
Finalize the Minimal Viable Product Features 

### ISSUE THREAD

**Issue by HarshCasper** [HarshCasper/AllNotes#9]
## Description 

Finalize the Minimal Viable Product Features. We received a lot of feedback on the potential product features, but we need to shortlist a certain set of features, which would be our baseline for development and deployment. All the Product Perspectives would be available over the Notion Workspace, so if you want to work on this Issue, take a look at them and decide the MVP Features. 

Document the set of features in a Markdown File and make a PR to the `docs/` directory with the filename as the `ROADMAP.md`. If you want further clarity on this Issue, feel free to drop me a message on Slack.

**Comment by HarshCasper** [comment]
@manikajain11 Can you convert the Markdown File to a Markdown File once I put apt comments and make a PR on the `docs` directory?

**Comment by manikajain11** [comment]
@HarshCasper yes sure. Actually I was learning how to convert the same and facing some issues. I'll add it positively once you finalize the comments.
